Source/WebCore/accessibility/AccessibilitySlider.cpp

```diff
@@ -171,7 +175,9 @@ LayoutRect AccessibilitySliderThumb::elementRect() const
     auto* sliderRenderer = dynamicDowncast<RenderSlider>(m_parent->renderer());
     if (!sliderRenderer)
         return LayoutRect();
-    if (auto* thumbRenderer = sliderRenderer->element().sliderThumbElement()->renderer())
+    CheckedRef element = sliderRenderer->element();
+    CheckedPtr sliderThumbElement = element->sliderThumbElement();
```

Comment:
Why use the Checked variants? Elements can be ref-counted

Reply:
Also this should use the `protected` or `checked` member functions rather than an extra local variable since there's only one usage of each in this method.